rules/linux/defense_evasion_vi_binary.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/03/03"
+maturity = "production"
+updated_date = "2022/03/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell.
+This activity is not standard use with this binary for a user or system administrator and could potentially indicate
+malicious actor attempting to improve the capabilities or stability of their access."
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via the vi command"
+references = ["https://gtfobins.github.io/gtfobins/vi/"]
+risk_score = 47
+rule_id = "89583d1b-3c2e-4606-8b74-0a9fd2248e88"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Defense Evasion", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id,process.pid with maxspan=1m
```

Comment:
can you verify that when doing this by `process.entity_id` (vs `host.id,process.pid`) that it is not working as expected? They match in the data provided (and we should use that when possible)

Reply:
process.entity_id currently does not work due to the reasons provided in the archived convo I sent you but this should be fixed in 8.2 with the endpoint POSIX process event data changes

Reply:
sequence by host.id and process.pid works perfectly fine right now and when the POSIX changes get introduced we will go back and test with process.entity_id and change to that